docs/docsite/rst/network/user_guide/platform_netconf_enabled.rst

```diff
@@ -84,6 +84,39 @@ Example NETCONF Task with configurable variables
      vars:
        ansible_private_key_file: /home/admin/.ssh/newprivatekeyfile
 
-Note: For nectonf connection plugin configurable variables .. _Refer: https://docs.ansible.com/ansible/latest/plugins/connection/netconf.html
+Note: For netconf connection plugin configurable variables see :ref:`netconf <netconf_connection>`.
+
+Bastion/Jumphost Configuration
+------------------------------
+To use a jump host to connect to a NETCONF enabled device you must set the ``ANSIBLE_NETCONF_SSH_CONFIG`` environment variable.
+
+``ANSIBLE_NETCONF_SSH_CONFIG`` can be set to either:
+  - 1 or TRUE (to trigger the use of the default SSH config file ~/.ssh/config)
+  - The absolute path to a custom SSH config file.
+
+The SSH config file should look something like: 
+
+.. code-block:: ini
+
+  Host *
+    proxycommand ssh -o StrictHostKeyChecking=no -W %h:%p jumphost-username@jumphost.fqdn.com
+    StrictHostKeyChecking no
+
+Authentication for the jump host must use key based authentication.
+
+You can either specify the private key used in the SSH config file:
+
+.. code-block:: ini
+
+  IdentityFile "/absolute/path/to/private-key.pem"
+
+Or you can use an ssh-agent.
+
+ansible_network_os auto-detection
+---------------------------------
+
+If ``ansible_network_os`` is not specified for a host, then Ansible will attempt to automatically detect what ``network_os`` plugin to use.
+
+``ansible_network_os`` auto-detection can also be triggered by using ``auto`` as the ``ansible_network_os``. (Note: Previously ``default`` was used instead of ``auto``).
```

Comment:
Do we need to worry about people upgrading to 2.9 and breaking things?

Reply:
@pabelanger  I don't believe so.

I think there are two scenarios we need to think about.

1. User specified no ansible_network_os. 
This will still trigger the auto-detection and the outcome will be the same, either the auto detection is successful and the correct network_os is used, or the auto-detection fails and default is uses
2. User specified ansible_network_os = default
Previously this would trigger the auto-detection (despite the user wanting to use the default network_os) which would fail and default would be used. After the change, the result will be the same, (but auto-detection will not have been tried).

I think the only scenario that could cause an issue is if someone was specifying ansible_network_os=default in order to trigger auto-detection (which I don't believe was documented anywhere)

Reply:
@pabelanger 
Agreed with @markafarrell 
We should be fine with this change.